docs/en_us/internal/testing.rst

```diff
@@ -70,6 +70,12 @@ UI Acceptance Tests
    write end-user acceptance tests directly in Python, using the
    framework to maximize reliability and maintainability.
 
+Internationalization Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Any new text that is added should be internationalized and translated.
```

Comment:
This section is confusing to me. I think this is mostly about automated testing, and we don't typically write tests for i18n, etc.